src/components/AttachmentModal.js

```diff
@@ -71,15 +72,17 @@ class AttachmentModal extends PureComponent {
 
         this.state = {
             isModalOpen: false,
-            isConfirmModalOpen: false,
+            isAttachmentInvalid: false,
+            attachmentInvalidReasonTitle: null,
```

Comment:
We should use empty String for attachmentInvalidReasonTitle because Confirm modal accepts string as title but we pass null which cause in this [issue](https://github.com/Expensify/App/issues/21947). Fixed it [here](https://github.com/Expensify/App/pull/22354) by providing empty String.